polling_stations/apps/data_finder/views.py

```diff
@@ -146,6 +159,10 @@ def get_context_data(self, **context):
         self.station = self.get_station()
         self.directions = self.get_directions()
 
+        context['has_election'] = self.has_election()
+        if not context['has_election']:
```

Comment:
sounds like a plan. If we do it that way in future, would it be sensible for that app to be a dependency shared with WDIV (so we aren't solving the same problem twice in 2 different places), or does WDIV necessarily obtain election info from YNR so it can be referenced against other the other YNR data?